### ISSUE TITLE
Change order in tagging window

### ISSUE THREAD

**Issue by genkimaps** [genkimaps/wildcam#65]
In the tagging window, “age” and “sex” should be directly above/below each other as well as “individual count” and “group count”; ideally order should be:

Identified by
Species
Count
Group Count
New sequence
Age
Sex
Behaviour
Coat colour
Has collar
Mark type
Tags/marks
Snow cover
Comments

**Comment by mitch-fen** [comment]
Might be related to #34

**Comment by genkimaps** [comment]
@mitch-fen Yes those are related, but different in implementation. This issue is just a cosmetic change; the other one is customizing the display on a per project basis.
